.github/workflows/comment-on-asciidoc-changes.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Comment on PR for .asciidoc changes
+
+on:
+  # We need to use pull_request_target to be able to comment on PRs from forks
+  pull_request_target:
+    types:
+      - synchronize
+      - opened
+      - reopened
+    branches:
+      - main
+      - master
+      - "9.0"
```

Comment:
we dont follow stack conventions branch name pattern in the RUM repo. Is this required?

Reply:
good call out. No, it is not